applications/GettingStartedWith/HtmlEditor/index.html

```diff
@@ -0,0 +1,53 @@
+<div id="html-editor">
+    <h2>
+        <img src="/Content/images/doc/21_1/htmlEditor/HtmlEditor.png" alt="HtmlEditor">
+        Formatted Text Editor (HTML Editor)
+    </h2>
+    <br>
+    <p>DevExtreme JavaScript HTML Editor is a client-side WYSIWYG text editor that allows its users to format textual and visual content and store it as HTML or Markdown.</p>
+    <p>Supported features:</p>
+    <ul>
+        <li>Inline formats:
+            <ul>
+                <li><strong>Bold</strong>, <em>italic</em>, <s>strikethrough</s> text formatting</li>
+                <li>Font, size, color changes (HTML only)</li>
+            </ul>
+        </li>
+        <li>Block formats:
+            <ul>
+                <li>Headers</li>
+                <li>Text alignment</li>
+                <li>Lists (ordered and unordered)</li>
+                <li>Code blocks</li>
+                <li>Quotes</li>
+            </ul>
+        </li>
+        <li>Custom formats</li>
+        <li>HTML and Markdown support</li>
+        <li>Mail-merge placeholders (for example, %username%)</li>
```

Comment:
Is there API to actually generate documents based on a mail-merge template? I mean I can add "%username%" to a document in Notepad. I don't need special features for that. We should emphasize API and data source support that will generate documents based on a template. "%username%" values should come from somewhere. If the data source doesn't have a "username" field, then I don't need such a placeholder. Or is there a limited number of placeholders and this feature doesn't work like a regular mail merge functionality that obtains values from the bound data source?

Reply:
This feature does require a data source, but I'm afraid I don't catch your drift. Do you mean that this item should read "API for mail-merge placeholders (for example, %username%)"?